it/full/spec/it_spec.sh

```diff
@@ -14,6 +14,9 @@ Describe "Hermit"
     It "creates hermit activation script in ./bin/activate-hermit"
       The file ./bin/activate-hermit should be exist
     End
+    It "creates intellij idea plugin configuration in ./.idea/externalDependencies.xml"
+      The file ./.idea/externalDependencies.xml should be exist
+    End
```

Comment:
@alecthomas Is this an OK place to add the integration test?

Reply:
Yep!